config.md

```diff
@@ -245,7 +245,7 @@ Presently there are `Prestart`, `Poststart` and `Poststop`.
 * [`Poststop`](#poststop) is a list of hooks to be run after the container process exits
 
 Hooks allow one to run code before/after various lifecycle events of the container.
-Hooks MUST be called in the listed order.
+Hooks MUST be called in the listed order if supported by the platform.
```

Comment:
On Tue, May 31, 2016 at 10:34:39AM -0700, Mrunal Patel wrote:
> > -Hooks MUST be called in the listed order.
> > +Hooks MUST be called in the listed order if supported by the platform.
>
> @wking If that is the intent, then we should clarify further whether
> the platform will error out or just not call the hooks.

I'm pretty sure “just not call the hooks” should not be valid runtime
behavior.  This is related to, but maybe not quite covered by, the “…
unable to create the environment specified…” language @cyphar has
in-flight with #397.